src/Nethermind/Nethermind.AuRa/Validators/ContractValidator.cs

```diff
@@ -139,13 +135,12 @@ public override void PreProcess(Block block, ProcessingOptions options = Process
                     bool reorganisationHappened = block.Number <= _lastProcessedBlockNumber;
                     if (reorganisationHappened)
                     {
-                        PendingValidators pendingValidators = null;
-                        if (!(block.Number <= CurrentPendingValidators?.BlockNumber))
-                        {
-                            pendingValidators = LoadPendingValidators();
-                        }
-
-                        SetPendingValidators(pendingValidators);
+                        SetPendingValidators(block.Number <= CurrentPendingValidators?.BlockNumber ? null : LoadPendingValidators());
```

Comment:
readability - maybe worth to name some named argument in the line before?
(that was exactly the type of comments you had in the past and I agree it is worth to introduce such improvements)